src/axom/primal/operators/evaluate_integral.hpp

```diff
@@ -212,6 +322,64 @@ double evaluate_area_integral(const primal::CurvedPolygon<T, 2> cpoly,
   return total_integral;
 }
 
+/*!
+ * \brief Evaluate an integral on the interior of a CurvedPolygon object.
```

Comment:
This says `CurvedPolygon` but the input is an array of NURBS curves.

We should probably mention that the result is only meaningful if the curves (approximately) enclose a region.